wordpress-mobile/WordPress-iOS
# Fix screenshot generation script

This PR fixes up the screenshot generation script to match the current NUX flow in the app. I also had to add some extra accessibility identifiers in various places, as the script was previously attempting to match on localised strings (so failed in non-English languages).

**To test:**

* Check you can successfully run tests on the `WordPressScreenshotGeneration` scheme
* You could also try running `snapshot` in the `Scripts/fastlane` directory
* Note that even though the script has been improved, it can still be quite temperamental – particularly if your internet connection is slow.
